pyrobosim/examples/test_world.py

```diff
@@ -0,0 +1,139 @@
+"""
+Unit tests for core world modeling
+"""
+import os
+import pytest
+import numpy as np
+
+from pyrobosim.core.world import World
+from pyrobosim.core.world import Room
+from pyrobosim.utils.pose import Pose
+from pyrobosim.core.world import Object
+from pyrobosim.core.world import Hallway
+
+from pyrobosim.utils.general import get_data_folder
+
+class TestWorldModeling():
+    """ Tests for the world modelling tools"""
+
+    @staticmethod
+    @pytest.mark.dependency()
+    def test_create_world_default():
+        """ Tests the creation of a world """
+        
+        TestWorldModeling.world = World()
+        TestWorldModeling.world.set_metadata(locations=os.path.join(get_data_folder(), "example_location_data.yaml"),
+                   objects=os.path.join(get_data_folder(), "example_object_data.yaml"))
+        assert TestWorldModeling.world is not None
+    
+    @staticmethod
+    @pytest.mark.dependency(depends=["TestWorldModeling::test_create_world_default"])
+    def test_create_room():
+        """ Tests the creation of a room """
+
+        room1_name = "kitchen"
+        room1_coords = [(-1, -1), (1.5, -1), (1.5, 1.5), (0.5, 1.5)]
+        room1 = Room(room1_coords,
+                    name=room1_name,
+                    color=[1, 0, 0])
+        TestWorldModeling.world.add_room(room1)
+
+        assert len(TestWorldModeling.world.rooms) == 1
+        assert TestWorldModeling.world.get_room_by_name(room1_name) == room1
+        
+        room2_name = "bedroom"
+        room2_coords = [(1.75, 2.5), (3.5, 2.5), (3.5, 4), (1.75, 4)]
+        room2 = Room(room2_coords,
+                    name=room2_name,
+                    color=[1, 0, 0])
+        TestWorldModeling.world.add_room(room2)
+
+        assert len(TestWorldModeling.world.rooms) == 2
+        assert TestWorldModeling.world.get_room_by_name(room2_name) == room2
+
+    @staticmethod
+    @pytest.mark.dependency(depends=
+        ["TestWorldModeling::test_create_world_default","TestWorldModeling::test_create_room"])
```

Comment:
Add space after commas. True here and in all subsequent dependency marks

```suggestion
        ["TestWorldModeling::test_create_world_default", "TestWorldModeling::test_create_room"])
```